src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "Views\GraphingCalculator\KeyGraphFeaturesPanel.g.h"
+#include "CalcViewModel\GraphingCalculator\EquationViewModel.h"
+#include "Controls/MathRichEditBox.h"
+#include "Controls/EquationTextBox.h"
+
+namespace CalculatorApp
+{
+    enum KeyGraphFeaturesFlag
+    {
+        Domain = 1,
+        Range = 2,
+        Parity = 4,
+        Periodicity = 8,
+        Zeros = 16,
+        YIntercept = 32,
+        Minima = 64,
+        Maxima = 128,
+        InflectionPoints = 256,
+        VerticalAsymptotes = 512,
+        HorizontalAsymptotes = 1024,
+        ObliqueAsymptotes = 2048,
+        MonotoneIntervals = 4096
+    };
+
+ public
+    ref class MonotonicityItem sealed
+    {
+    public:
+        property Platform::String ^ Expression;
+        property Platform::String ^ Direction;
+    };
+
+     public
+    ref class ExpressionItem sealed
+    {
+    public:
+        property Platform::String ^ Expression;
+    };
+
+public
+    ref class KeyGraphFeaturesPanel sealed : public Windows::UI::Xaml::Data::INotifyPropertyChanged
+    {
+    public:
+        KeyGraphFeaturesPanel();
+
+        OBSERVABLE_OBJECT_CALLBACK(OnPropertyChanged);
+        OBSERVABLE_PROPERTY_R(Platform::String ^, Domain);
```

Comment:
All of these observable properties already exist in the EquationViewModel. This control should only take a ViewModel and bind to the properties on the view model. The ViewModel can handle the logic to return an error message or not, the control should just cleanly bind to each property.

Reply:
I can move the logic. and then just bind directly.